src/components/Restoration/RestorationType.vue

```diff
@@ -14,12 +14,16 @@
             v-model="selectRestorationType"
             @change="changeRestorationType()"
           >
-            <v-radio id="full-radio-button" label="Full Restoration" :value=RestorationTypes.FULL />
+            <v-radio
+              class="radio-button"
+              id="full-radio-button"
+              label="Full Restoration"
+              :value=RestorationTypes.FULL />
```

Comment:
Next time, I'd prefer to see the closing bracket on the next line (in line with the opening bracket).